File: 2310817120009_AlysaArmelia_PemrogramanI_Modul2/Python/PRAK205-2310817120009-AlysaArmelia.py
import math
a,b=input("").split()
c= round(math.sqrt(int(b) * int(b) - int(a) * int(a)))
k= round(int(b) + int(a) + int(c))
l= round((int(a) * int(c))/2)
print(f'Alas = {c} cm')
print(f'Tinggi = {a} cm')
print(f'Keliling = {k} cm')
print(f'Luas = {l} cm^2\n')

File: 2310817120009_AlysaArmelia_PemrogramanI_Modul2/Bahasa C/PRAK205-2310817120009-AlysaArmelia.c
#include <stdio.h>
#include <math.h>

int main()
{
    int a;
    int b;
    int c;
    int k;
    int l;
    scanf("%d %d", &a, &b);
    c=sqrt(pow(b,2)-pow(a,2));
    k=a + b + c;
    l=0.5*c*a;
    printf("Alas = %d cm\n", c);
    printf("Tinggi = %d cm\n", a);
    printf("Keliling = %d cm\n", k);
    printf("Luas = %d cm^2\n\n", l);
    return 0;
}